www/scripts/productlist/productlist.js

```diff
@@ -19,18 +19,21 @@ function (declare, topic, domConstruct, Button, BorderContainer,
 
     //---------------------------- Global objects ----------------------------//
 
+    API_VERSION = '6.0'
```

Comment:
why defined here in productlist.js and not in a central js file like version.js?